pypesto/petab/importer.py

```diff
@@ -304,6 +304,25 @@ def create_prior(self) -> NegLogParameterPriors:
         else:
             return None
 
+    def create_startpoint_method(self):
+        """
+        Creates a startpoint method, if the PEtab problem specifies an
+        initializationPrior. Returns None, if no initializationPrior
+        is specified.
+        """
+        if petab.INITIALIZATION_PRIOR_TYPE \
+                not in self.petab_problem.parameter_df:
+            return None
+
+        def startpoint_method(n_samples: int, **kwargs):
```

Comment:
@jvanhoefer : shouldn't that be `n_starts`? Seems like that is what's expected in `pesto.startpoint.util.assign_startpoints`.

Reply:
yep